spec.md

```diff
@@ -598,6 +598,8 @@ expressed in the short form.
   - `service_healthy`: specifies that a dependency is expected to be "healthy"
     (as indicated by [healthcheck](#healthcheck)) before starting a dependent
     service.
+  - `service_completed_successfully`: specifies that a dependency is expected to run 
+    to successful completion before starting a dependent service.
```

Comment:
Apologies if this is overly pedantic, but could we specify that `successful completion` means exiting with code `0`?

Reply:
we discussed this point during compose-spec meeting and prefered **not** to get into this Linux process implementation detail. Compose model could be applied to distinct things (let's say, VMs, unikernels or WASM payload) which might have a distinct way to notify the platform execution has been successful.

Reply:
Fair point, resolving 🙂